src/utils/Validator.ts

```diff
@@ -182,22 +180,13 @@ const validateTimestamp = (timestamp: Date | number): void => {
     throw new InvalidTimestampError(`Timestamp ${String(timestamp)} is invalid`);
   }
 
-  const isTooOld = validator.isBefore(
-    timestampStr,
-    new Date(Date.now() - Constants.MAX_TIMESTAMP_PAST_AGE).toISOString(),
-  );
-  const isTooNew = validator.isAfter(
-    timestampStr,
-    new Date(Date.now() + Constants.MAX_TIMESTAMP_FUTURE_AGE).toISOString(),
-  );
-
-  if (isTooOld) {
+  if (timestampStr < new Date(Date.now() - Constants.MAX_TIMESTAMP_PAST_AGE).toISOString()) {
```

Comment:
Why are you using `toISOString` and not unix time?

Reply:
I just reused the logic from the existing code. `toISOString` was there in the existing code as well.

Reply:
Let me change the code to compare Date with Date instead of string with string.